src/ontology/external/nord.robot.tsv

```diff
@@ -6999,3101 +7000,4 @@ MONDO:0011312					http://purl.obolibrary.org/obo/mondo#NORD_LABEL	http://purl.ob
 MONDO:0011305					http://purl.obolibrary.org/obo/mondo#NORD_LABEL	http://purl.obolibrary.org/obo/mondo#nord_rare		MONDO:NORD
 MONDO:0011308					http://purl.obolibrary.org/obo/mondo#NORD_LABEL	http://purl.obolibrary.org/obo/mondo#nord_rare		MONDO:NORD
 MONDO:0011303					http://purl.obolibrary.org/obo/mondo#NORD_LABEL	http://purl.obolibrary.org/obo/mondo#nord_rare		MONDO:NORD
-MONDO:0011304					http://purl.obolibrary.org/obo/mondo#NORD_LABEL	http://purl.obolibrary.org/obo/mondo#nord_rare		MONDO:NORD
```

Comment:
@matentzn are you aware of any changes from NORD resulting in the removal of all of these llines?

Reply:
Per Sabrina's comment, I will contact NORD to confirm this change is expected.

Reply:
NORD responded that this data change is correct and may also change in future NORD data releases.